tests/grid/grid_tools_find_closest_vertex_02.cc

```diff
@@ -0,0 +1,78 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2020 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+// Check find_closest_vertex_of_cell when a mapping is present
+
+#include <deal.II/dofs/dof_handler.h>
+
+#include <deal.II/fe/fe_q.h>
+#include <deal.II/fe/fe_system.h>
+#include <deal.II/fe/mapping_q_eulerian.h>
+
+#include <deal.II/grid/grid_generator.h>
+#include <deal.II/grid/grid_out.h>
+#include <deal.II/grid/grid_tools.h>
+#include <deal.II/grid/tria.h>
+
+#include "../tests.h"
+
+
+template <int dim, int spacedim>
+void
+test(const Point<spacedim> &p, double displacement)
+{
+  deallog << "dim: " << dim << ", spacedim: " << spacedim
+          << " with displacement " << displacement << std::endl;
+
+  Triangulation<dim, spacedim> tria;
+  GridGenerator::hyper_cube(tria);
+  tria.refine_global();
+
+  // Vector fe
+  FESystem<dim, spacedim>   fe{FE_Q<dim, spacedim>(1), spacedim};
+  DoFHandler<dim, spacedim> dh(tria);
+  dh.distribute_dofs(fe);
+
+  // Displacement vector.
+  Vector<double> disp(dh.n_dofs());
+  for (unsigned int i = 0; i < dh.n_dofs(); ++i)
+    disp[i] = displacement;
+
+  // mapping
+  MappingQEulerian<dim, Vector<double>, spacedim> mapping(1, dh, disp);
+
+  for (const auto cell : tria.active_cell_iterators())
+    {
+      const auto i =
+        GridTools::find_closest_vertex_of_cell<dim, spacedim>(cell, p, mapping);
+      const auto &v = mapping.get_vertices(cell);
+
+      deallog << "Closest vertex of cell " << cell << " to " << p
+              << " is cell->vertex(" << i << ") :" << v[i] << std::endl;
```

Comment:
Can you add a space here?
```suggestion
              << " is cell->vertex(" << i << ") : " << v[i] << std::endl;
```